src/components/CreateArticleForm.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import axios from "axios";
+import {Form, Input, TextArea, Button, Message } from 'semantic-ui-react'
+
+const CreateArticleForm = () => {
+  const [message, setMessage] = useState();
+  const [title, setTitle] = useState();
+  const [subtitle, setSubtitle] = useState();
+  const [content, setContent] = useState();
+
+  let headers = useSelector((state) => state.currentUser);
+  const saveArticle = async (e) => {
+    e.preventDefault();
+    setTitle(e.target.title.value);
+    setSubtitle(e.target.input_sub_title.value);
+    setContent(e.target.input_content.value);
+    setMessage("Your article was created");
+    headers = {
+      ...headers,
+      "Content-type": "application/json",
+      Accept: "application/json",
+    };
+    try {
+      await axios.post(
+        "/articles",
+        {
+          article: {
+            title: title,
+            sub_title: subtitle,
+            content: content,
+          },
+        },
+        {
+          headers: headers,
+        }
+      );
+    } catch (err) {
+      console.error(err);
+      alert("Something went wrong!");
```

Comment:
use the error messages that you get from the backend

Reply:
Yes we set response.data.message instead